### DISCUSSION TITLE
Ability to edit custom timers

### DISCUSSION THREAD

**Discussion by TheAngryRogue** [smasherprog/EqTool#42] [Ideas]
A lot of mobs in different zones have zone timers that differ from the actual zone timer. My current find is the citizen faction dwarves that druids and wizards quad in BB. One set has a respawn of 12 minutes. The ones at the docks are 22 minutes. Custom timer is set for 10 minutes. 
Instead of making you do all the timer editing; a right click -> Edit Mob Timer;  option would be helpful. I know the P99 wiki shows mob timers for individual mobs. It is not shown in the MobInfo window, but would be easier to edit manually having it.

**Comment by smasherprog** [comment]
Just add the name names / zones / respawn times here and ill add them. The outliers i have to add manually, but its ez to do

**Reply by CtrlAltComplete** [reply]
Vhalen Nostrolo in South Karana is a 6min timer. Its currently 6:40 with timer in eqtool

**Comment by CtrlAltComplete** [comment]
How do I add custom timers to the trigger window?

**Comment by smasherprog** [comment]
Zone timers fixed, thanks for bringing this up

**Comment by smasherprog** [comment]
type

start timer customertimer 20

**Comment by smasherprog** [comment]
I think this request was made because timers were incorrect. Since they have been fixed, this shouldnt be an issue any more
